### PR TITLE
schema_generator

### PR DESCRIPTION
**Pull request by Mugetsu42069**

### CODE REVIEW

File: Bipintask4.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import json
+import os
+import glob
+
```
Comment (jonscyr) [inline Bipintask4.py:4]: wrap these in functions. have a main function which calls other functions.

give a brief description on what this script is doing at the top of the file as a 2 or 3 line comment

**Review by jonscyr — CHANGES_REQUESTED** [review]

File: Bipintask4.py (hunk 1 above)
Comment (Mugetsu42069) [reply Bipintask4.py:6]: made the changes

File: Bipintask4.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
+# This script reads JSON files containing schema information, generates JSON schemas
```
Comment (jonscyr) [inline Bipintask4.py:1]: bad file name, follow shaurya's file structure.

**Review by jonscyr — CHANGES_REQUESTED** [review]

File: Bipintask4.py (hunk 2 above)
Comment (Mugetsu42069) [reply Bipintask4.py]: done

**Review by jonscyr — APPROVED** [review]

**Pull request merged**